src/pages/index.js

```diff
@@ -23,14 +49,58 @@ const addNewCardFormValidator = new FormValidation(
 editProfileFormValidator.enableValidation();
 addNewCardFormValidator.enableValidation();
 
-const cardList = new Section(
-  { items: initialCards, renderer: createCard },
-  ".gallery"
+const areYouSureModal = new ModalWithForm("modal-box_type_delete-card");
+const areYouSureModalValidation = new FormValidation(
+  document.querySelector(".form__delete-card"),
+  config
+);
+areYouSureModalValidation.enableValidation();
+
+const updateAvatarIcon = document.querySelector(".edit-avatar-button");
+const updateAvatarModal = new ModalWithForm("modal-box_update-avatar", () => {
+  updateProfilePicture(updateAvatarModal.getInputValues());
+});
+const updateAvatarFormValidator = new FormValidation(
+  document.querySelector(".modal-box_update-avatar", config),
+  config
 );
-cardList.renderItems();
+updateAvatarFormValidator.enableValidation();
 
 function createCard(cardObject) {
-  const card = new Card(cardObject, "#card", previewCard);
+  const card = new Card(
+    cardObject,
+    "#card",
+    previewCard,
+    {
+      handleDelete: () => {
+        areYouSureModal.open();
+        areYouSureModal.changeEventListener(() => {
+          console.log(cardObject, cardObject._id);
+          api.deleteCard(cardObject._id).then((res) => {
+            card.deleteCard();
+            areYouSureModal.close();
+            areYouSureModal.eliminateEventListener();
+          });
+        });
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**